### PR TITLE
Add missing packages from T001-22 to README

### PR DESCRIPTION
**Pull request by dominiquesydow**
## Description
Add missing packages from T001-22.

## TODOs
- [x] README

## Status
- [x] Ready to go

### CODE REVIEW

**Comment by dominiquesydow** [comment]
@jaimergp I went through the talktorials and updated the external packages list.

Shall I add 
- `google.colab`
- `condacolab`
Those are not part of the environment, is that intentional?

**Comment by jaimergp** [comment]
Those are installed only if running on Colab. Normal envs should not use it.

**Comment by dominiquesydow** [comment]
@jaimergp, can I merge from your side?

File: README.md [hunk 1]
```diff
@@ -121,20 +121,37 @@ It will help measure the impact of the TeachOpenCADD platform and future funding
   [`mdanalysis`](https://www.mdanalysis.org/),
   [`biopython`](https://biopython.org/),
   [`biopandas`](http://rasbt.github.io/biopandas/),
-  [`opencadd`](https://opencadd.readthedocs.io/en/latest/)
+  [`opencadd`](https://opencadd.readthedocs.io/en/latest/),
+  [`plip`](https://github.com/pharmai/plip),
+  [`openff`](https://github.com/openforcefield/openff-toolkit),
```
Comment (jaimergp) [inline README.md:126]:
```suggestion
  [`openff-toolkit`](https://github.com/openforcefield/openff-toolkit),
  [`openmmforcefields`](https://github.com/openmm/openmmforcefields),
```

We had missed this!

**Comment by jaimergp** [comment]
@dominiquesydow Go ahead after adding my suggestion!

**Pull request merged**